Packages/MIES/MIES_DataConfigurator.ipf

```diff
@@ -2347,11 +2347,13 @@ static Function DC_AddEpochsFromStimSetNote(panelTitle, channel, stimset, stimse
 						// shift all flipped pulse intervalls by pulseDuration to the left, except the rightmost with pulseNr 0
 						if(!pulseNr)
 							subEpochBegin = epochEnd - startTimes[0] - pulseDuration
-							subEpochEnd = subEpochBegin + pulseDuration
+							// assign left over time after the last pulse to that pulse
+							subEpochEnd = epochEnd
 						else
-							subEpochEnd = epochEnd - startTimes[pulseNr - 1] - pulseDuration
+							subEpochEnd = epochEnd - startTimes[limit(pulseNr - 1, 0, inf)] - pulseDuration
```

Comment:
Why is the limit added here?
The else part of the if check can only be run with pulseNr > 0.

Reply:
Good point. Will remove.